release-doc-in-process/all_vf.TTL

```diff
@@ -218,31 +226,30 @@ vf:committedCategory
 #        rdfs:domain         vf:ExchangeAgreement ;
 #        rdfs:range          vf:ExchangeTaxonomyItem .
 
-vf:parent
-        a                   owl:ObjectProperty ;
-        rdfs:label          "parent" ;
-        rdfs:domain         vf:ResourceTaxonomyItem ;
-        rdfs:range          vf:ResourcetaxonomyItem ;
-        rdfs:comment        "A more general or broader category of resource." .
+#vf:parent
+#        a                   owl:ObjectProperty ;
+#        rdfs:label          "parent" ;
+#       rdfs:domain         vf:ResourceTaxonomyItem ;
+#        rdfs:range          vf:ResourcetaxonomyItem ;
+#        rdfs:comment        "A more general or broader category of resource." .
 
-vf:involvesResource
+vf:involves
         a                   owl:ObjectProperty ;
-        rdfs:label         "involves resource" ;
+        rdfs:label         "involves" ;
         rdfs:domain         vf:Commitment ; # create one of these for intents
         rdfs:range          vf:EconomicResource ;
         rdfs:comment        "EconomicResource the Commitment intends or commits to affect. Could be an actual resource or a category or type of resource." .
 
 vf:underlyingResource
         a                   owl:ObjectProperty ;
         rdfs:label         "underlying resource" ;
-        rdfs:domain         vf:Resource ;
-        rdfs:range          vf:Resource ;
+        rdfs:domain         vf:EconomicResource ;
+        rdfs:range          vf:EconomicResource ;
         rdfs:comment        "A more concrete resource upon which a resource is based." .
 
 vf:relationship  a          owl:ObjectProperty ;
```

Comment:
domain is what class(s) it is a property for; range is what it is in the way of an object (think of like a foreign key)